erpnext_com/www/docs/user/manual/en/loan-management/loan-application.md

```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
```

Comment:
global: use realistic names and not Test Customer, Test Company. Company name: Unico Plastics
List the features as explained in the page format: https://github.com/frappe/erpnext/wiki/Page-format-for-ERPNext-docs
for example, in loan application Loan Info, Loan Security Details, Repayment Info will be the 'Features'